signatures_test.go

```diff
@@ -0,0 +1,70 @@
+package cobblerclient
+
+import (
+	"testing"
+
+	"github.com/ContainerSolutions/go-utils"
+)
+
+func TestGetSignatures(t *testing.T) {
+	c := createStubHTTPClient(t, "get-signatures-req.xml", "get-signatures-res.xml")
+
+	err := c.GetSignatures()
+	utils.FailOnError(t, err)
+}
+
+func TestGetValidBreeds(t *testing.T) {
+	c := createStubHTTPClient(t, "get-valid-breeds-req.xml", "get-valid-breeds-res.xml")
+
+	res, err := c.GetValidBreeds()
+	utils.FailOnError(t, err)
+	if len(res) <= 0 {
+		t.Fatalf("Expected a lenght of greater then one.")
+	}
```

Comment:
This should be <= 1 according to the error message. Also there is a typo in the message

Same problem in the following tests